communication/mailing-list-guidelines.md

```diff
@@ -0,0 +1,201 @@
+- [Delete the post].
+- In some cases you might need to ban a user from the group, follow
+  [these instructions] on how stop a member from being able to post to the group.
+  For more technical help on how to use Google Groups, check the [Groups Help]
+  page.
+
+
+### Moderator expectations and guidelines
+
+Moderators should adhere to the general Kubernetes project
+[moderation guidelines].
+
+
+#### New user posting queue
+
+New members who post to the Mailing list will automatically have their messages
+put in the [moderation queue]. Moderators of the list will receive a
+notification of their message and should process them accordingly.
+
+
+#### Annual permissions review
+
+SIG and WG Moderators must establish an annual review of their Mailing lists
+to ensure their Moderator list is current and includes [Mailing List owners].
+Many of the SIG and WG Mailing lists pre-date current communication policy and
+an annual review ensures ownership is up to date.
+
+This review does not need to occur at a specific recurring date and can be
+combined with other actions such as SIG/WG leadership changes or sub-project
+additions.
+
+
+---
+
+## Mailing list creation
+
+Create a Google Group at https://groups.google.com/forum/#!creategroup,
+following the below procedure:
+- Each SIG must have two discussion groups with the following settings.
+  - `kubernetes-sig-<foo>` (the discussion group):
+    - Anyone can view content.
+    - Anyone can join.
+    - Moderate messages from non-members of the group.
+    - Only members can view the list of members.
+  - `kubernetes-sig-<foo>-leads` (list for the leads, to be used with Zoom and
+    Calendars)
+    - Only members can view group content.
+    - Anyone can apply to join.
+    - Moderate messages from non-members of the group.
+    - Only members can view the list of members.
+- Groups should be created as e-mail lists with at least three owners and must
+  include the [Mailing list owners](#mailing-list-owners).
+-  To add the owners, visit the **Group Settings** (drop-down menu on the right
+   side), select **Direct Add Members** on the left side and add them via their
+   email address (with a suitable welcome message).
+- In **Members/All Members** select the [Mailing list owners] and assign them
+  to the **owner role**.
+- Set the following permissions to **Public**:
+  - **View topics**
+  - **Post**
+  - **Join the Group**
+- Create and share your _"meeting notes"_ Google doc with the following
+  permissions settings:
+  - **Can edit** for members of the newly created Mailing List.
+  - **Can comment** for `kubernetes-dev@googlegroups.com`
+  - **View only** for anyone with the link. **NOTE:** Depending on
+    employer organization policy, this may not be possible to configure. The
+    document should be copied over to an account without the restriction and 
+    include the owner reference at the top of the document.
```

Comment:
this conflicts with guidance at https://github.com/kubernetes/community/blob/master/communication/zoom-guidelines.md#setting-up-your-meeting-and-moderation to not make meeting information available anonymously

Reply:
agreed and known. we are doing a major overhaul of all of our comms docs this week. this is going to be changed when all sigs are on to the new zoom processes (you are already). PR in flight today.